upi/aws/cloudformation/02_cluster_infra.yaml

```diff
@@ -4,9 +4,18 @@ Description: Template for Openshift Cluster UPI Network Elements (Route53 & LBs)
 Parameters:
   ClusterName:
     AllowedPattern: ^([a-zA-Z][a-zA-Z0-9\-]{0,31})$
+    MaxLength: 32
+    MinLength: 1
     ConstraintDescription: Cluster name must be alphanumeric, start with a letter and a maximum of 32 characters
     Description: A short, representative cluster name to use for hostnames, etc.
     Type: String
+  InfrastructureName:
+    AllowedPattern: ^([a-zA-Z][a-zA-Z0-9\-]{0,31})$
+    MaxLength: 32
+    MinLength: 1
+    ConstraintDescription: Infrastructure name must be alphanumeric, start with a letter and a maximum of 32 characters
+    Description: A short, unique cluster ID used to tag cloud resources and identify items owned/used by the cluster.
```

Comment:
The value of InfrastructureName should be matched with the output of `jq -r .infraID metadata.json`, but not any string randomly specified by user, right? If yes, is it possible to make the description more clear to emphasize this point.